### PR TITLE
windows buildbot updates

### PR DESCRIPTION
**Pull request by shuffle2**
* remove debug-win-x64 builder
* remove signbin.bat invocation
* add release-win-arm64 builder

### CODE REVIEW

File: buildbot/master.cfg [hunk 1]
```diff
@@ -249,7 +242,7 @@ def make_dolphin_win_build(build_type, arch, mode="normal"):
                                          "HASH": WithProperties("%s", "revision"),
                                          "AUTHOR": WithProperties("%s", "author"),
                                          "DESCRIPTION": WithProperties("%s", "description"),
-                                         "TARGET_SYSTEM": "Windows x86" if arch == "x86" else "Windows x64",
+                                         "TARGET_SYSTEM": "Windows arm64" if arch == "arm64" else "Windows x64",
```
Comment (PEmu2) [inline buildbot/master.cfg:245]: Maybe a bit pedantic but should arm be capitalized here? Microsoft appears to use either "ARM64" or "Arm64" in documentation.

Comment (shuffle2) [reply buildbot/master.cfg:245]: maybe? tbh I'm not sure if this is user-facing field or not.

Comment (shuffle2) [reply buildbot/master.cfg:245]: shrug i'll just capitalize it in some more places

Comment (PEmu2) [reply buildbot/master.cfg]: Oh yeah I was just thinking in user facing text. If this doesn't appear on the website feel free to ignore.

**Pull request merged**